### DISCUSSION TITLE
Moving ./config inside ./alire?

### DISCUSSION THREAD

**Discussion by mosteo** [alire-project/alire#812] [General]
@Fabien-Chouteau , this arose the other day on gitter. You'll have a clearer picture:

- Are the contents of `./config` always regenerated from scratch? (I think so).
- If so, could this conceivably be moved to `./alire/config`? So not to create an extra artifact in the user crate.

We are putting it in `.gitignore` already, so it would be also ignored inside `./alire`. And it could still be forced into version control if wanted.

**Comment by Fabien-Chouteau** [comment]
> * Are the contents of `./config` always regenerated from scratch? (I think so).

Yes. 

> * If so, could this conceivably be moved to `./alire/config`? So not to create an extra artifact in the user crate.

Even if they are generated by Alire, I am not sure it's a good idea to hide them away. In particular because you cannot compile the project without those generated files. So some people might want to check them in if they want the project to be usable outside of Alire.

**Reply by mosteo** [reply]
Right, the later is certainly an important point in favor of keeping it as-is.
